frontend/src/components/student-course/components/modules-cards-container/components/module-card/styles.module.scss

```diff
@@ -1,6 +1,7 @@
 .card {
   display: flex;
-  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
   padding-left: 15px;
```

Comment:
On the screenshot it looks like module card has different padding from left and right sides, I think they should be equal